packages/psd/src/classes/Group.ts

```diff
@@ -68,4 +70,30 @@ export class Group implements NodeBase<NodeParent, NodeChild> {
   get id(): number | undefined {
     return this.layerFrame?.id;
   }
+
+  async userMask(): Promise<Uint8Array | undefined> {
+    const userMask = this.layerFrame?.userMask;
+
+    if (!userMask) {
+      return undefined;
+    }
+    const {maskData} = this;
+
+    if (!maskData) {
```

Comment:
Why not combine both ifs like in `realUserMask`?

Reply:
done